src/resolve/resolved-manager.h

```diff
@@ -164,6 +167,8 @@ int manager_start(Manager *m);
 
 uint32_t manager_find_mtu(Manager *m);
 
+int send_dns_notification(Manager* m, DnsAnswer* answer, const char* query_name);
```

Comment:
… also here. Please use the same style as surrounding code.

Reply:
Addressed in latest patch